eth/eventhandler/event_handler_test.go

```diff
@@ -449,37 +540,132 @@ func TestHandleBlockEventsStream(t *testing.T) {
 		})
 	})
 
-	// Receive event, unmarshall, parse, check parse event is not nil or with an error,
-	// public key is correct, owner is correct, operator ids are correct
-	t.Run("test ValidatorRemoved event handle", func(t *testing.T) {
-		_, err = boundContract.SimcontractTransactor.RemoveValidator(
-			auth,
-			validatorData1.masterPubKey.Serialize(),
-			[]uint64{1, 2, 3, 4},
-			simcontract.CallableCluster{
-				ValidatorCount:  1,
-				NetworkFeeIndex: 1,
-				Index:           1,
-				Active:          true,
-				Balance:         big.NewInt(100_000_000),
-			})
-		require.NoError(t, err)
-		sim.Commit()
+	t.Run("test ValidatorRemoved event handling", func(t *testing.T) {
+		// Must throw error "malformed event: could not find validator share"
```

Comment:
Where is this checked?

Reply:
After @olegshmuelov PR merging we do actually check this via key manager